documentation/modules/ref-address-example-forwarders.adoc

```diff
@@ -39,9 +40,9 @@ spec:
 <2> (Required) Specifies the remote address to forward messages to. The address must be prefixed with the connector name and must be identical to the address matching patterns defined on the connector.
 <3> (Required) Specifies the direction of message flow, which is either `out` or `in`. A value of `out` forwards messages to the remote endpoint. A value of `in` forwards messages from the remote endpoint.
 
-== Forwarding messages from a remote endpoint to a local queue
+== Forwarding messages from a remote AMQP server to a local queue
 
-In this example, the remote endpoint has an address `prices/milk` that we can receive messages from and move to `myqueue`.
+In this example, the remote AMQP server has an address `prices/milk` that we can receive messages from, and then move to `myqueue`.
```

Comment:
Suggest this change to write in the second person and slightly improve readability:

"In this example, you receive messages from an address `prices/milk` on a remote AMQP server. You then move the messages to a local queue, `myqueue`."

Reply:
@lulf, Is this suggestion technically correct? (In particular I'm wondering about the second sentence, since the messages are moved by the forwarder, so maybe: "The messages are then moved to a local queue, `myqueue`.")

Reply:
Yes, LGTM!